src/en/send.md

```diff
@@ -94,61 +94,106 @@ You can leave out this argument if your service only has one reply-to email addr
-1. Go to the __Templates__ page and select the relevant email template.
-1. Select __Edit__.
-1. Add a placeholder to the email template using double brackets. For example:
+You are able to control how files are delivered to recipients on every
+API call. GC Notify cannot access or decrypt your files.
 
-"Download your file at: ((link_to_file))"
+::: tip Choosing a sending method
 
-### Upload your file
+People generally expect files to be directly attached to emails. With that in mind, it’s not uncommon to see attachments being blocked by security rules or email providers. Use the unique link method to prevent your attachments from being blocked.
 
-::: tip File types and size requirements
-You can upload PDF, CSV, .jpeg, .png, .odt, .txt, .rtf,  Microsoft Excel and Microsoft Word Document files. Your file must be smaller than 10MB.
+Before choosing a sending method, perform tests to see what works best for your use case.
 
-[Contact us](https://notification.canada.ca/contact) if you need to send other file types.
 :::
 
-You’ll need to convert the file into a string that is base64 encoded.
+### Upload your file
+
+To send files, pass a dictionary of arguments in the `personalisation` argument. Pass this dictionary to the placeholder key if it’s present in your template or use a name of your choice.
+
+You’ll need to specify:
 
-Pass the encoded string into an object with a `file` key, and put that in the personalisation argument. For example:
+- `file`: convert the file into a string that is base64 encoded. Example: `Q2FuYWRh` (`Canada` encoded in base64)
+- `filename`: the filename of the file you are sending. Example: `service_name_applicant_name.pdf`
+- `sending_method`: specify how you want to send this file. Either `attach` for the direct attachment method or `link` to generate a unique link
+
+#### If you’re sending files as direct attachments
+
+Specify `attach` as `sending_method`.
+
+For example:
+
+**Template**
+```
+Hello ((first_name)),
+
+We received your application on ((application_date)).
+
+You will find your application attached.
+```
 
+**HTTP parameters**
 ```json
 "personalisation": {
   "first_name": "Amala",
   "application_date": "2018-01-01",
-  "link_to_file": {
+  "application_file": {
     "file": "file as base64 encoded string",
-    "filename": "your_custom_filename.pdf"
+    "filename": "your_custom_filename.pdf",
+    "sending_method": "attach"
   }
 }
 ```
 
-**CSV files**
+#### If you’re sending files with unique links
+
+1. Add a placeholder to the email template
+1. Send HTTP requests, specify `link` as `sending_method`
+
+**Add a placeholder to the template**
+
+1. [Sign in to GC Notify](https://notification.canada.ca/sign-in).
+1. Go to the __Templates__ page and select the relevant email template.
+1. Select __Edit__.
+1. Add a placeholder to the email template using double brackets. For example:
+
+```
+You can [now download your application](((link_to_file))).
```

Comment:
Is it actually triple brackets?

Reply:
This is how it is at the moment yeah according to the templates documentation https://notification.canada.ca/templates

Not really confusing for devs, could be for others, agreed